examples/workflow/requirements.txt

```diff
@@ -1 +1,2 @@
 dapr-ext-workflow>=0.1.0
+aiohttp>=3.9.4 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
Instead we should pin a higher minimum version of `dapr-ext-workflow` which itself should bring a newer version of the core `dapr` Pyhthon SDK. `aiohttp` is introduced via the core `dapr` SDK. That's where we should be updating it and release a new version if necessary.

Could you please do that instead? I'm not a fan of overloading it here.

Reply:
Also sign your commits please